docs/extensions/shoot-health-status-conditions.md

```diff
@@ -1,14 +1,16 @@
 # Contributing to Shoot Health Status Conditions
 
 Gardener checks regularly (every minute by default) the health status of all shoot clusters.
-It categorizes its checks into four different types:
+It categorizes its checks into five different types:
 
 * `APIServerAvailable`: This type indicates whether the shoot's kube-apiserver is available or not.
 * `ControlPlaneHealthy`: This type indicates whether the core components of the Shoot controlplane (ETCD, KAPI, KCM..) are healthy.
 * `EveryNodeReady`: This type indicates whether all `Node`s and all `Machine` objects report healthiness.
 * `ObservabilityComponentsHealthy`: This type indicates whether the  observability components of the Shoot control plane (Prometheus, Vali, Plutono..) are healthy.
 * `SystemComponentsHealthy`: This type indicates whether all system components deployed to the `kube-system` namespace in the shoot do exist and are running fine.
 
+In case of workerless shoot `EveryNodeReady` condition is not present on shoot since there are no nodes in the cluster.
```

Comment:
```suggestion
In case of workerless `Shoot`, `EveryNodeReady` condition is not present in the `Shoot`'s conditions since there are no nodes in the cluster.
```